extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -28,16 +28,21 @@ define('wikia.articleVideo.featuredVideo.tracking', [], function () {
 			throw new Error('No tracking label provided');
 		}
 
-		var finalGAData = {
+		var trackingData = {
 			action: gaData.action || 'click',
 			category: gaCategory,
 			label: gaData.label,
 			//value tracks sound state: 1 for muted, 0 for unmuted
-			value: Number(playerInstance.getMute())
+			value: Number(playerInstance.getMute()),
+
```

Comment:
@jcellary We would propose to use (default) trackingevent as an eventName for tracking all video-related events, is that fine with you?

Reply:
I think that's fine, but could you paste me the full url that you'd be sending? (e.g. what you see in chrome on your devbox under networking) I just need to make sure we can easily filter them out by a short set of criteria.

Reply:
```
https://beacon.wikia-services.com/__track/special/trackingevent?c=509&x=harrypotter&a=60661&lc=en&n=0&u=0&s=oasis&beacon=ThisIsFake&cb=24728&videoId=FYykS9se&player=jwplayer&ga_action=impression&ga_category=featured-video&ga_label=load&ga_value=1
```
would be an exemplary request